README.md

```diff
@@ -51,6 +52,37 @@ val subscriber = kafka.publish(ProducerProperties(
 Source(publisher).map(_.toUpperCase).to(Sink(subscriber)).run()
 ```
 
+#### Java
+```Java
+private ActorSystem actorSystem = ActorSystem.create("ReactiveKafka")
+private String brokerList = "localhost:9092";
+private String zooKeeperHost = "localhost:2181";
+  
+private ReactiveKafka kafka = new ReactiveKafka()
+
+ConsumerProperties<String> cp = new PropertiesBuilder.Consumer()
+        .withZooKeeperHost(zooKeeperHost)
+        .withBrokerList(brokerList)
+        .withGroupId(groupName)
+        .withTopic(topic)
+        .withStringDecoder()
+        .build();
+
+Publisher<String> publisher = kafka.consume(cp, actorSystem);
+        
+ProducerProperties<String> pp = new PropertiesBuilder.Producer()
+        .withZooKeeperHost(zooKeeperHost)
+        .withBrokerList(brokerList)
+        .withClientId(groupName)
+        .withTopic(topic)
+        .withStringEncoder()
+        .build();
+
+Subscriber<String> subscriber = kafka.publish(pp, actorSystem);
+
+Source(publisher).map((d) -> d.toUpperCase()).to(Sink(subscriber)).run()
```

Comment:
Can you replace `((d) -> d.toUpperCase())` with `(String::toUpercase())`?

Suppose it doesn't matter for the docs.